### PR TITLE
New skills.html

### PR DESCRIPTION
**Pull request by Hage714**

### CODE REVIEW

**Comment by Elegido7** [comment]
code well done

File: skills.html [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+    <!-- Right -->
+    <div>
+      <a href="" class="me-4 text-reset">
+        <i class="fab fa-facebook-f"></i>
+      </a>
+      <a href="" class="me-4 text-reset">
+        <i class="fab fa-twitter"></i>
+      </a>
+      <a href="" class="me-4 text-reset">
+        <i class="fab fa-google"></i>
```
Comment (VioletWarutere) [inline skills.html:129]: code looks good

**Comment by Cokumujr** [comment]
code is okay

**Comment by kisinja** [comment]
let's continue downloading dollars 😅

**Comment by infinityblip** [comment]
not so bad ..tueendele kujituma

**Comment by Davidmugece** [comment]
The code is perfect

**Comment by yussufnoor1234** [comment]
perfectly done.

**Comment by Dmunyi** [comment]
good code.

**Pull request merged**